lib/src/controls/navigation/tree_view.dart

```diff
@@ -146,6 +146,10 @@ class TreeViewItem with Diagnosticable {
   /// item is going to be one of the selected items
   bool? selected;
 
+  /// Whether the current item is disabled.
+  ///
+  bool? disabled;
```

Comment:
I believe this can be non-nullable.

Reply:
after add ```disabled: true```, the TreeViewItem will not be selected, and the son TreeViewItem can still be selected.

![image](https://github.com/bdlukaa/fluent_ui/assets/14890680/0a5903f8-2931-402f-a93d-03ad7ae31f5d)

![image](https://github.com/bdlukaa/fluent_ui/assets/14890680/d99e740c-bb50-4190-8a90-a6afd51578fa)